generator/net_snmp.go

```diff
@@ -126,7 +126,7 @@ var (
 	}
 )
 
-// Initilise NetSNMP. Returns MIB parse errors.
+// Initialise NetSNMP. Returns MIB parse errors.
```

Comment:
Didn't we say we were going with US spellings?

Reply:
@brian-brazil +1
I can go for it, and replace every `Initialise` with  `Initialize`. Should I?

Reply:
Yes please